app/search.php

```diff
@@ -0,0 +1 @@
+<?php require 'functions.php'; ?>
```

Comment:
Does this reference need to be in all the partial files if its in the `header.php` already?